src/python/pants/engine/internals/scheduler_integration_test.py

```diff
@@ -11,7 +12,7 @@ def test_visualize_to():
     # Tests usage of the `--native-engine-visualize-to=` option, which triggers background
     # visualization of the graph. There are unit tests confirming the content of the rendered
     # results.
-    with temporary_dir() as destdir:
+    with temporary_dir(root_dir=os.getcwd()) as destdir:
```

Comment:
Did you forget to clean this up / why is where the tmpdir is rooted now important when it wasn't before?

Reply:
There has ~always been validation of `file_option` paths, but for some reason it wasn't triggering here before.